src/adr/articles/0002-verifiable-credentials.md

```diff
@@ -0,0 +1,112 @@
+
+This could be an optional addition to the existing Use a LPA service to allow for a wider demographic of users.
+
+It needs to be fully interopable with
+
+1. Future technology such as mobile phone digital wallets
+2. Industry standard validation and authenticity implementations
+3. Potential for other agencies to use as a proof of identity
+
+We would need to be able to reliably sign the credential for others to validate the authenticity.
+
+We would need the Verifiable Credential to be able to store the appropriate required information for the consumer to understand its responsibilities.
+
+## Technical
+
+### Interoperability - How does this enable the exchange of information
+
+The Verifiable Credentials data model is a [W3C Open Standard](https://www.w3.org/TR/vc-data-model/). As long as the standards are adhered to, interopability will be high.
+
+The signing of a Verifiable Credential should also be standards driven and supported by the wider industry.
+
+### Developer Knowledge - How well known is this in our current skill sets
+
+**Overall**: 2/10
+The data model is well defined and an understood pattern for representing data.
+
+The validity and authenticity aspect of the specification are early in discovery.
+
+If signing was done via a certificate provider setting, such is done for SSL Certification, then the understanding will be higher. It will come with extra challenges on management of the signing keys and the key life cycle.
+
+### Support/Open Source - Is it well supported
+
+The Verifiable Credentials data model is a [W3C Open Standard](https://www.w3.org/TR/vc-data-model/).
+
+The implementation of verifying and signing Verifiable Credentials is covered in the standard. This is in early stages of research and development with various companies such as [RSA Labs and Project Mercury](https://mercury.rsalabs.com/).
+
+### Scalability
+
+The credential is a small file that can be stored anywhere. Either as a file on a storage platform such as Cloud Object Storage, database, Personal Data Store or a users device.
+
+Once created and digitally signed, it is handed over to the Citizen and only requires checks on its validity from us as its issuer.
+
+## Ethics
+
+### Mitigate against being tech deterministic
+
+TBD
+
+### Ensure you conduct inclusive research
+
+TBD
+
+### Think big and imagine what the impact of your work can be
+
+TBD
+
+### Interrogate your data decisions
+
+TBD
+
+### Decision
+
+We should continue to explore the idea.
+
+We should continue to consult other parties to ensure we work to the open standards and maintain the ability to pivot in this direction as the industry integrates into this technology.
+
+### Consequences
+
+Depending on further research this will vary.
+
+Using the method of RSA Labs Project Mercury would mean us setting up the Office of the Public Guardian or the Ministry of Justice as an issuer.
```

Comment:
I don't see what Project Mercury is actually going to offer over creating the credentials ourselves